test-tx-meta-baseline-next/InvokeHostFunctionTests.json

```diff
@@ -29,728 +29,6 @@
 		21
 	],
 	"Native stellar asset contract" : [ "bKDF6V5IzTo=", "i7kalOBorks=" ],
-	"Soroban authorization" : 
```

Comment:
Looks like some Soroban tests don't have the `[tx]` label, but meta was still generated from them. This should be figured out before the PR is merged.